input-profile.js

```diff
@@ -0,0 +1,164 @@
+WL.registerComponent(
+	"input-profile",
```

Comment:
For indentation please use 4 spaces instead of tabs like in the other components 🙏

Reply:
configured with prettier